pinc/RoundDescriptor.inc

```diff
@@ -175,6 +180,36 @@ class Round extends Stage
     function is_a_mentee_round() { return !is_null($this->mentor_round); }
     function is_a_mentor_round() { return !is_null($this->mentee_round); }
 
+    // -----------
+
+    function has_a_daily_page_limit()
+    {
+        return !is_null($this->daily_page_limit);
+    }
+
+    // XXX This probably doesn't belong here, but I'm not sure where it does.
```

Comment:
Yeah, not sure where is the best place for this. It seems odd to have it here. It might make more sense to have it in `User` and pass in the round.

Reply:
I looked at putting it in `User`, that didn't feel right either.

I carved the guts of it off to a TallyBoard method; that seemed to make sense.

But then the function itself I just stuck in a new `inc` file.

Reply:
Roger that. 👍